src/AutoRest.CSharp/Common/AutoRest/Plugins/NewProjectScaffolding.cs

```diff
@@ -399,9 +398,12 @@ private string GetBrandedTestCSProj()
             }
             writer.ProjectReferences.Add(new($"..\\src\\{Configuration.Namespace}.csproj"));
             // add the package references
-            foreach (var package in _brandedTestDependencyPackages)
+            if (!Configuration.AzureArm)
```

Comment:
If we take a look at https://github.com/Azure/azure-sdk-for-net/blob/f67a51ae48ae0628733b1cf207d6fec4b0db12ac/eng/Directory.Build.Common.targets#L262-L268, there is no `Azure.Identity`.
So, for a new branded test project will it be missing `Azure.Identity` if we don't add it to test csproj for mgmt?

Reply:
since we have test project dependencies for mgmt in azure-sdk-for-net, we should remvoe them here.
Please check another comment regarding `Azure.Identity`.

Reply:
`Azure.Identity` is not always needed for SDK test projects. So in scaffolding, we will not add it by default. When a special SDK need this, it can manually added.